voicevox_engine/utility/path_utility.py

```diff
@@ -2,18 +2,43 @@
 import sys
 import traceback
 from pathlib import Path
+from typing import Literal
 
 from platformdirs import user_data_dir
 
 
+def runtime_type() -> Literal["nuitka", "pyinstaller", "python"]:
```

Comment:
良い切り分けですね！
細かいのですが、`_`付けてプライベート関数化してもいいかも？
```suggestion
def _runtime_type() -> Literal["nuitka", "pyinstaller", "python"]:
```